ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,97 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{sl_assessments_port | default("7201")}} 
+
+IMPROVEMENT_PROJECT_BASE_URL
+#Application ENV example local/dev/production
+NODE_ENV={{sl_assessments_node_env}}
+
+#Log configuration
+#Enable console logging
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+
+#Enable writing logs to a file
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+
+#service based URL
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+SHIKSHALOKAM_USER_PROFILE_FETCH_ENDPOINT={{sl_assessments_shikshalokam_user_profile_fetch_endpoint | default("/api/user/v1/read")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+SHIKSHALOKAM_BASE_HOST={{sl_assessments_shikshalokam_base_host}}
```

Comment:
Why are we having variables named shikshalokam @aks30 ?

Reply:
@rjshrjndrn we will remove this.

Reply:
shikshalokam variables is removed